modules/kubectl/init.zsh

```diff
@@ -0,0 +1,40 @@
+#
+# Provides 'kubectl' aliases and utiities.
+#
+# Authors:
+#   Bruno Miguel Custodio <brunomcustodio@gmail.com>
+#
+
+# Return if requirements are not found.
+if (( ! ${+commands[kubectl]} )); then
+  return 1
+fi
+
+# Enable completion for 'kubectl'.
+cache_file="${0:h}/cache.zsh"
+kubectl_command="${commands[kubectl]}"
+
+if [[ "${kubectl_command}" -nt "${cache_file}" || ! -s "${cache_file}" ]]; then
```

Comment:
@indrajitr if I understand correctly, this serves as a means to check whether the completion cache file is up-to-date (as different tool versions could generate different completion files).